ambari-server/src/main/resources/Ambari-DDL-Derby-CREATE.sql

```diff
@@ -96,25 +96,6 @@ CREATE TABLE configuration_base (
   CONSTRAINT PK_configuration_base PRIMARY KEY (id)
 );
 
-CREATE TABLE clusterconfig (
```

Comment:
Should the "clusterconfig" table really be removed with this change? 

This seems like something that should be modified in a follow-up patch, especially considering the possible affects this could have on the rest of Ambari (UI, etc).

Reply:
The purpose of this change is to add/modify the tables necessary for storing multi-instance blueprints. I think removing cluster config should be tracked by a separate effort, considering the cascading effects on the code and tests. Also not sure if it is not a general multi-everything related task rather than blueprint-specific.

Reply:
This table existed twice in some DDL's so I removed the duplicate.

Reply:
ok, thanks for clarifying this.